components/vf-navigation/vf-navigation.js

```diff
@@ -0,0 +1,47 @@
+// vf-navigation
+// only required for vf-navigation--on-this-page
+
+/**
+ * Function for JS scroll to top functionality
+ * That must be executed exactly once
+ * @example vfNavigationOnThisPage()
+ */
+export function vfNavigationOnThisPage() {
+  var sectionList = document.querySelectorAll("[data-vf-js-navigation-on-this-page-container='true'],[data-vf-js-navigation-on-this-page-container='1']")[0];
+  var section = document.querySelectorAll("[data-vf-js-navigation-on-this-page]");
+  var sectionPositions = {};
+  var i = 0;
+
+  if (!sectionList || !section) {
+    // exit: either sections or section content not found
+    return;
+  }
+  if (sectionList.length == 0 || section.length == 0) {
+    // exit: either sections or section content not found
+    return;
+  }
+
+  Array.prototype.forEach.call(section, function(e) {
```

Comment:
No reason to still use `Array.prototype.forEach.call()` instead of `section.forEach()`, its supported on IE9+

Reply:
thanks for the tip, old habits die hard.